### PR TITLE
Op 762 handle 401 api errors

### PR DESCRIPTION
**Pull request by tsognong**
At that ime I just redirect a user to login page when the token expires

### CODE REVIEW

**Comment by axelpezzo** [comment]
@almeida1492 seems ok to me, what do you think?

File: src/libraries/authUtils/tokenHasExpired.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export interface JwtTokenModel {
+  exp: number;
+  sub: string;
+  auth: string;
+}
+
+export const tokenHasExpired = (token: any): boolean => {
+  const { exp } = jwtDecode<JwtTokenModel>(token);
+  const expirationTime = exp * 1000 - 60000;
```
Comment (axelpezzo) [inline src/libraries/authUtils/tokenHasExpired.ts:11]: I would move this in a constant

Comment (alefalezza) [reply src/libraries/authUtils/tokenHasExpired.ts:11]: @SteveGT96 can you do that?

Move "60000" to a constant `TOKEN_EXPIRATION_TIMEOUT` in `src\consts.ts`

**Comment by SteveGT96** [comment]
OK Alessandro !

File: src/libraries/authUtils/tokenHasExpired.ts (hunk 1 above)
Comment (SteveGT96) [reply src/libraries/authUtils/tokenHasExpired.ts]: @alefalezza It's done !

**Comment by axelpezzo** [comment]
@almeida1492 can we go ahead with this MR?

**Review by almeida1492 — APPROVED** [review]

**Comment by almeida1492** [comment]
> @almeida1492 can we go ahead with this MR?

I think so.

**Pull request merged**